minicluster/pom.xml

```diff
@@ -46,6 +46,11 @@
       <artifactId>curator-test</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>compile</scope>
```

Comment:
why compile scope rather than test scope?

Reply:
This way, ~MultiProcessCluster` (In `src/`, similar to `LocalAlluxioCluter`) can extend `TestRule`. I think this makes it more convenient to use

Reply:
ack